### ISSUE TITLE
Question about training dataset

### ISSUE THREAD

**Issue by create7859** [qinzheng93/GraphSCNet#4]
Hello, 

thank you for sharing your research.

My question is, just like the CAPE dataset could be used for training, is it possible to create a mesh sequence-based Dynamic FAUST dataset as a training dataset?

**Comment by qinzheng93** [comment]
Yes, I think FAUST can be used for training but it has to be processed into point clouds.
